code/firmware/rosco_m68k_firmware/stage2/load/include/elf.h

```diff
@@ -0,0 +1,123 @@
+/*
+ *------------------------------------------------------------
+ *                                  ___ ___ _
+ *  ___ ___ ___ ___ ___       _____|  _| . | |_
+ * |  _| . |_ -|  _| . |     |     | . | . | '_|
+ * |_| |___|___|___|___|_____|_|_|_|___|___|_,_|
+ *                     |_____|       firmware v2
+ * ------------------------------------------------------------
+ * Copyright (c)2020-2022 Thomas Jager
+ * See top-level LICENSE.md for licence information.
+ *
+ * Definitions for ELF file contents.
+ * Only contains the ELF Header and Program Header.
+ * 
+ * Structures, names, and values based on:
+ *   Tool Interface Standard (TIS)
+ *   Executable and Linking Format (ELF) Specification
+ *   Version 1.2
+ * ------------------------------------------------------------
+ */
+
+#ifndef __ROSCO_M68K_ELF_H
+#define __ROSCO_M68K_ELF_H
+
+#include <stdint.h>
+
+#pragma pack(push, 1)
```

Comment:
Minor; _Generally_ I prefer `__attribute__((packed))` on the individual structures, as that's where I tend to look (less cognitive load than having to scroll around and see what's been pushed / popped).